### ISSUE TITLE
concept:prepared

### ISSUE THREAD

**Issue by dshorthouse** [tdwg/attribution#32]
### prepared
|  |  |
| ---- | ---- |
| **Definition** | Made the subject ready for use |
| **Existing concept** |  |
| **Existing namespace** |  |
| **Existing concept identifier** |  |
| **Format** | string |
| **Examples** | "prepared" |
| **Notes** | |

**Comment by Cubey0** [comment]
Leading space needed?

**Comment by dshorthouse** [comment]
@Cubey0 In the table you mean? Yeah, an annoying artifact of not having (nor wanting) column headers in markdown. There probably is a way to remedy this, but I'm lazy.

**Comment by RBGE-Herbarium** [comment]
What are we including in here? Are we including mounting and stuffing? Are we also including preparing for other actions, eg loans? Are we including conserving here?

**Comment by qgroom** [comment]
I read this as mounting, stuffing etc
If you want to include loans, perhaps we should have a `packedBy` and `shippedBy` term

**Comment by RBGE-Herbarium** [comment]
Would it be "Prepared the object"? or even "Prepared the physical object"?

**Comment by wouteraddink** [comment]
I think the agent is the subject here and what is prepared is the object? If I read this as mounting, stuffing etc and to exclude thinks like packed for shipping, maybe better rename it as madeObjectPreparation/madeSpecimenPreparation, or is that too long? Definition could then be something like "Made an object preparation to conserve the object as a specimen".

**Comment by wouteraddink** [comment]
or createdPreparationFromObject

**Comment by dshorthouse** [comment]
@wouteraddink Indeed, the intent here is to represent all the (unspecified) actions involved in treating an object with the aim of conserving it. Any action in the vocab requires an agent that executes an action and an object that receives the action. And so, it's redundant (and a regression) to specify this in the term itself – to a degree, it would re-expose the possibility for conflation/ambiguity in existing terms like `recordedBy` or `identifiedBy` if this weren't an item in a controlled vocabulary.

**Comment by wouteraddink** [comment]
Ok. The issue is that the term preparation for me is more specific a conservation term and prepare  as verb is not, but that may be caused by my limited English skills. Regarding your previous edit: I have never seen a good definition of a specimen..still looking for it. I thought that every natural object catalogued and curated for scientific study is called a specimen, including catalogued material samples?

**Comment by dshorthouse** [comment]
@wouteraddink Totally agree that the verb prepared is peculiar. However, it’s already being used by data publishers in overloaded recordedBy values. Here’s one of many, many examples: https://www.gbif.org/occurrence/1054693698. From what I’ve seen, it’s been primarily occurrences about vertebrates.

**Comment by matdillen** [comment]
Would it make sense to change this to `preserved`? This would not cover packing loans, but I don't think we'd want that. Using `prepared` implies something the object needs to be prepared for (preservation, loans, measurement, decontamination...), which may not always be clear and we don't have a field to offer that kind of information.

**Comment by dshorthouse** [comment]
@tucotuco @Jegelewicz tagging you both on this one because I often seen "Prepared" in vertebrate specimen records as a prefix to recordedBy, many originating from Arctos. Should we retain `prepared` like this but with better definition or is the term itself poor and is better expressed as `preserved` or other?

**Comment by Jegelewicz** [comment]
We have the agent role "preparator" for this case.

See https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcollector_role

**Comment by Jegelewicz** [comment]
Transaction type roles are here

https://arctos.database.museum/info/ctDocumentation.cfm?table=cttrans_agent_role

**Comment by Jegelewicz** [comment]
I think prepared is an appropriate term - the person who does the preservation has been traditionally referred to as the "preparator" - at least that is my experience.

> I thought that every natural object catalogued and curated for scientific study is called a specimen, including catalogued material samples?

We have had many discussions about this because we also catalog cultural objects. We are in the process of removing all references to "specimen" in Arctos to be replaced with "cataloged item" or "object". See https://github.com/ArctosDB/arctos/issues/772

**Comment by dshorthouse** [comment]
Thanks @Jegelewicz. We have a different notion of `role` - that played on a particular entity rather than association with an agent so as not to conflate it with one's role in an institution. I see that your definition of the term is just as tautological as ours. What we're seeking is a clear definition of `prepared`.

**Comment by dshorthouse** [comment]
Suggestion for definition: Made the subject ready for use and/or longterm care.

**Comment by Jegelewicz** [comment]
Agree. Honestly, I think most curators and collection managers just assume they know what "preparation" means.

Google "specimen preparation" and you immediately get a bunch of stuff related to microscopy. I think this is a really good question:

> Would it make sense to change this to preserved?

However, it seems like a large cultural shift. But the basic definitions may make the choice easier.

  Prepared - made ready for use.

  Preserved - maintain (something) in its original or existing state.

**Comment by dshorthouse** [comment]
+1 @Jegelewicz I like those two separate terms and definitions. Preserved would accommodate actions taken long after an object is prepared.

**Comment by dshorthouse** [comment]
Made a new Action term, `preserved` #40

**Comment by deepreef** [comment]
Just a tangential response to this from @Jegelewicz :

> We have had many discussions about this because we also catalog cultural objects. We are in the process of removing all references to "specimen" in Arctos to be replaced with "cataloged item" or "object".

We are on a similar course, and after coming to realize several years ago that the cultural/anthropological/archaeological objects fit *remarkably* well within the DwC set of classes and terms, we will be applying the DwC class & term MaterialSample to both biological and non-biological objects.  It's a slight stretch of the term and its definition, but functionally/informatically, it allows us to elegantly converge biological and non-biological collection objects.

Apologies for the digression from the primary topic. To get it back on point, I'll also say I prefer the term 'prepared' in this context, and agree with the two separate terms & definitions as suggested by @Jegelewicz
